deps/curl/lib/http_chunks.c

```
@@ -257,7 +257,7 @@ static CURLcode httpchunk_readwrite(struct Curl_easy *data,
 
         if(tr) {
           size_t trlen;
-          result = Curl_dyn_addn(&ch->trailer, (char *)STRCONST("\x0d\x0a"));
+          result = Curl_dyn_addn(&ch->trailer, STRCONST("\x0d\x0a"));
```

Cppcheck (2.10-2)

[error] failed to expand 'Curl_dyn_addn', Wrong number of parameters for macro 'Curl_dyn_addn'.

(preprocessorErrorDirective)